pkg/pki/k8scsrpki/k8scsr_user.go

```diff
@@ -344,3 +346,53 @@ func (c *k8sCSR) Approve(ctx context.Context, signingReq *certsigningreqv1.Certi
 
 	return nil
 }
+
+func (c *k8sCSR) getCAChain(ctx context.Context, signingReq *certsigningreqv1.CertificateSigningRequest, certs []*certutil.CertificateContainer) ([]byte, error) {
+	var caChain []byte
+	signerName := strings.Split(signingReq.Spec.SignerName, "/")
+
+	if signerName[0] == v1alpha1.CertManagerSignerNamePrefix {
+		clusterIssuer := &certv1.ClusterIssuer{}
+		clusterIssuerName := signerName[1]
```

Comment:
Even if the signer name is `{{signingReq.Spec.SignerName}}/` by accident, `getCAChain` won't be triggered because `cert-manager` can't issue a certificate with a wrong signerName. So, I can't imagine the case.

Reply:
Yeah but we are relying on the fact that some outside factor deems it unlikely code ever gets here.

But if it does get here because
A, cert-manager lets this through in the future or
B, we refactor something and this accidentally gets called
this will result in a panic that could easily have been avoided.

I would rather see this fixed before merging, but I can let this go if you think it should go in as is.

Reply:
Whatever, I guess I can fix it once it's in if it only bothers me.